kore/src/Kore/Step/Simplification/Data.hs

```diff
@@ -811,8 +819,9 @@ applicationAxiomSimplifier applicationSimplifier =
         -> TermLikeSimplifier
         -> BuiltinAndAxiomSimplifierMap
         -> TermLike variable
+        -> Predicate variable
```

Comment:
Why do you have the extra argument here if you're not using it (esp. since this is a local definition)?

Reply:
`helper` needs this signature in order to be wrapped in a `BuiltinAndAxiomSimplifier`